src/systems/tracked-controls.js

```diff
@@ -7,6 +7,7 @@ var utils = require('../utils');
  * It maintains a list with the available tracked controllers
  */
 module.exports.System = registerSystem('tracked-controls', {
+  schema: {alwaysAllowIdPrefixes: {type: 'array'}},
```

Comment:
according to https://developer.mozilla.org/en-US/docs/Web/API/Gamepad
pose doesn't even exist, so i think some defensiveness may be appropriate